datasetinsights/io/downloader/base.py

```diff
@@ -48,6 +49,9 @@ def create_downloader(source_uri, **kwargs):
 
 
 class DatasetDownloader(ABC):
+    def __init__(self, **kwargs):
```

Comment:
Should be removed. This is not required.

Reply:
I'll remove it from here

Reply:
Actually, from the comments in `HTTPDownloader`, we should keep this to catch all arguments that was passed to constructor.

Reply:
I'll keep it here then and remove it from the other http_downloader